content/installation/python/PythonConfig.md

```diff
@@ -0,0 +1,51 @@
+<!-- 
+title: "Python Agent Configuration"
+description: "Configuring the Python Agent and Service"
+tags: "installation python django flask pyramid agent service configuration"
+-->
+
+The Python agent and Go service use a YAML file to alter the agent behavior. 
+
+## Load Path
+
+The configuration file is called *contrast_security.yaml* wherever it's located. The Python agent loads the configuration YAML from the following paths in order of precedence:
+
+1. The current working directory (e.g., *./contrast_security.yaml*)
+2. A subdirectory called *config*, (e.g., *./config/contrast_security.yaml*)
+3. Within the server's *etc* directory (e.g., */etc/contrast_security.yaml*)
+
+
+## General Configuration Options
+
+The configuration YAML consists of four sections. The agent and service may share a common configuration file, but only some options and sections are applicable to each process.
+
+* `contrast`: Options for locating and communicating with the Contrast interface Dashboard
+  * `url`: URL to connect to the Contrast application
+  * `api_key`: Organization's API key
+  * `service_key`: 
+  * `user_name`: 
```

Comment:
Do we need descriptions for these two items? If not, we should remove the colons.